bytes.md

```diff
@@ -0,0 +1,50 @@
+---
+title: BYTES
+toc: true
+---
+
+## Description
+
+The `BYTES` [data type](data-types.html) stores binary strings of variable, unlimited length.
```

Comment:
And if 64MB (or is that MiB? And it probably needs to be reduced by the overhead of the key) becomes the hard limit, I would suggest that it should probably be recommended to use less than half of that (32MB) to allow ranges to be split into at least two non-empty ranges.

And there may be practical reasons to keep individual row/column values even smaller, like below 1 MB. Large BLOB support is a distinct beast from normal transactional records.